From 4269591c4a59adb08cb00bd2dfad31c7869c9751 Mon Sep 17 00:00:00 2001
From: samaga Prasanna Krishna <samaga.krishna@intel.com>
Date: Fri, 7 Dec 2012 19:47:47 +0530
Subject: [PATCH 15/21] [CTP-Offload] Adding metadata info for gapless playback support

BZ: 58535

This patch defines Gapless metadata info for offload in audio.h.

Change-Id: I48c969a0708d336c7eb1ca3a5ce45a77570d8ac2
Signed-off-by: samaga Prasanna Krishna <samaga.krishna@intel.com>
Reviewed-on: http://android.intel.com:8080/79867
Reviewed-by: Kandasamy, Muthukumar <muthukumar.kandasamy@intel.com>
Reviewed-by: Anand, Jerome <jerome.anand@intel.com>
Reviewed-by: Hasan, MohammedX <mohammedx.hasan@intel.com>
Tested-by: Gupta, ArvindX K <arvindx.k.gupta@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 include/hardware/audio.h |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/include/hardware/audio.h b/include/hardware/audio.h
index 9db16c7..66b592c 100644
--- a/include/hardware/audio.h
+++ b/include/hardware/audio.h
@@ -145,6 +145,8 @@ __BEGIN_DECLS
 #define AUDIO_OFFLOAD_CODEC_ENCODE_OPTION "music_offload_encode_option"
 #define AUDIO_OFFLOAD_CODEC_NUM_CHANNEL  "music_offload_num_channels"
 #define AUDIO_OFFLOAD_CODEC_DOWN_SAMPLING  "music_offload_down_sampling"
+#define AUDIO_OFFLOAD_CODEC_DELAY_SAMPLES  "delay_samples"
+#define AUDIO_OFFLOAD_CODEC_PADDING_SAMPLES  "padding_samples"
 
 /**************************************/
 
-- 
1.7.4.1

